src/solve.jl

```diff
@@ -314,6 +314,7 @@ function init{algType<:OrdinaryDiffEqAlgorithm,recompile_flag}(
   if initialize_integrator
     initialize_callbacks!(integrator, initialize_save)
     initialize!(integrator,integrator.cache)
+    save_start && typeof(alg) <: CompositeAlgorithm && copyat_or_push!(alg_choice,1,integrator.alg.current_alg)
```

Comment:
That's already done https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/617302fb6097a01f172211081c5b581c12a4d627/src/solve.jl#L203